rest/taskana-rest-spring-example/pom.xml

```diff
@@ -52,21 +52,48 @@
 			<artifactId>h2</artifactId>
 			<version>1.4.197</version>
 		</dependency>
-
+		<dependency>
+			<groupId>pro.taskana</groupId>
+			<artifactId>taskana-web</artifactId>
+			<version>${project.version}</version>
+		</dependency>
 		<!-- Tests -->
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
-
 	<build>
 		<plugins>
 			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
+				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
What is this for?
I don't think this is necessary since taskana-web already packs everything in /static